sovrin_node/server/node_authn.py

```diff
@@ -10,6 +10,7 @@ class NodeAuthNr(NaclAuthNr):
     def __init__(self, ledger: Ledger):
         self.ledger = ledger
 
+    # TODO: This is not right, what if verkey changes?
```

Comment:
It will just go over the pool ledger and this happens only in case of succssfull code upgrade but yes, it can be improved